easybuild/easyconfigs/c/CGAL/CGAL-4.8.1-foss-2015a.eb

```diff
@@ -0,0 +1,38 @@
+name = 'CGAL'
+version = '4.8.1'
+gmpver = '6.0.0a'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'strict': True}
+
+# note: source URL needs to be updated for a new version (checksums too)!
+source_urls = ['https://github.com/%(name)s/%(namelower)s/releases/download/releases%2F%(name)s-%(version)s']
+sources = [SOURCE_TAR_XZ]
+checksums = ['1cbcb3f251822766ed3d744f923c782d']
+
+dependencies = [
+    ('Boost', '1.60.0'),
+    ('GMP', gmpver),
+    ('MPFR', '3.1.2', '-GMP-%s' % gmpver),
+]
+
+configopts  = "-DCMAKE_VERBOSE_MAKEFILE=ON "
+configopts += "-DBoost_INCLUDE_DIR=$EBROOTBOOST/include "
+configopts += "-DBoost_LIBRARY_DIRS=$EBROOTBOOST/lib "
+configopts += "-DBoost_SYSTEM_LIBRARY=$EBROOTBOOST/libboost_system.%s " % SHLIB_EXT
+configopts += "-DBoost_THREAD_LIBRARY=$EBROOTBOOST/libboost_thread.%s " % SHLIB_EXT
+configopts += "-DMPFR_INCLUDE_DIR=$EBROOTMPFR/include "
+configopts += "-DMPFR_LIBRARIES=$EBROOTMPFR/lib/libmpfr.%s " % SHLIB_EXT
+configopts += "-DGMP_INCLUDE_DIR=$EBROOTGMP/include "
+configopts += "-DGMP_LIBRARIES=$EBROOTGMP/lib/libgmp.%s " % SHLIB_EXT
+configopts += "-DGMP_LIBRARIES_DIR=$EBROOTGMP/lib "
+
+builddependencies = [
```

Comment:
@valtandor existing CGAL easyconfigs include `Eigen` as a build dep, but not here?